spec/sidekiq/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -158,6 +160,30 @@ def expect_retryable_error(error_class)
             )
           end
 
+          context 'when using LH Benefits Claims API instead of EVSS' do
+            before do
+              Flipper.enable(:disability_compensation_lighthouse_claims_service_provider)
+              allow_any_instance_of(BenefitsClaims::Configuration).to receive(:access_token).and_return('access_token')
+            end
+
+            after { Flipper.disable(:disability_compensation_lighthouse_claims_service_provider) }
+
+            let(:open_claims_cassette) { 'lighthouse/benefits_claims/index/claims_with_single_open_disability_claim' }
+
+            it 'logs the expected data for EP 400 merge eligibility' do
```

Comment:
This new Rspec scenario essentially tests for the same EP400 Merge behavior that we expect from using EVSS, but with the LH feature flag turned on, and a LH VCR cassette swapped in for the EVSS one.